x-pack/legacy/plugins/maps/common/constants.js

```diff
@@ -5,12 +5,20 @@
  */
 
 export const EMS_CATALOGUE_PATH = 'ems/catalogue';
+
 export const EMS_FILES_CATALOGUE_PATH = 'ems/files';
 export const EMS_FILES_DEFAULT_JSON_PATH = 'ems/files/file';
+export const EMS_GLYPHS_PATH = 'ems/fonts';
```

Comment:
Consider removing plurality since everything else is in the singular: `EMS_GLYPH_PATH`

Reply:
Actually, scratch that. I was looking at the next line with `EMS_SPRITE_PATH`. Maybe make that one plural then (`EMS_SPRITES_PATH`). Just so long as we're consistent